packages/react/src/index.ts

```diff
@@ -1,16 +1,21 @@
 import { createElement } from 'react';
 
 import type { CompiledJSX } from './jsx/jsx-local-namespace';
-import type { CssFunction, CSSProps, CssType } from './types';
 
-export type { CSSProps, CssFunction, CssType };
-
-export { keyframes } from './keyframes';
-export { styled } from './styled';
 export { ClassNames } from './class-names';
+export { createStrictAPI } from './create-strict-api';
 export { default as css } from './css';
 export { default as cssMap } from './css-map';
-export { createStrictAPI } from './create-strict-api';
+export { keyframes } from './keyframes';
+export { styled } from './styled';
+export type {
+  CSSProperties,
+  CSSProps,
+  CSSPseudos,
+  CssFunction,
+  CssType,
+  StrictCSSProperties,
+} from './types';
 export { type XCSSAllProperties, type XCSSAllPseudos, type XCSSProp, cx } from './xcss-prop';
```

Comment:
While we are here I noticed `XCSSAllProperties` doesn't type check when passed to the `XCSSProp` from `createStrictAPI`. Can you update it to be sourced from `StrictCSSProperties` instead of `CSSProperties`.

Reply:
Not catching what you mean—I don't think we're using `CSSProperties` in the output of `createStrictAPI`, right?  Don't see anything in https://github.com/atlassian-labs/compiled/blob/34674ae1a005c4bf2d4b227504b22e5b64ffe669/packages/react/src/create-strict-api/index.ts#L147

Reply:
Yeah exactly — this line https://github.com/atlassian-labs/compiled/blob/master/packages/react/src/create-strict-api/index.ts#L147 is a subst of `CSSProperties` so it type errors if you try and do:

```
XCSSProp<XCSSAllProperties>
```

Reply:
I got the context now; done in 2a93b6c2.